basilisk/src/ast/stack.c

```
@@ -0,0 +1,66 @@
+#include "stack.h"
+
+struct _Stack {
+  void * p;
+  int n, size;
+  void * (* push) (Stack *, void *);
+  void * data;
+};
+
+Stack * stack_new (int size)
+{
+  Stack * s = calloc (1, sizeof (Stack));
```

Clang (14.0.6)

[warning] variable name 's' is too short, expected at least 3 characters

(readability-identifier-length)